stable/superset/superset-init

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I don't see any relation to the mentioned pull-secret for this.
Was it added by mistake?

Reply:
oops :) mea culpa, removed.